Classes/Core/RequestHandler.php

```diff
@@ -121,12 +123,12 @@ protected function checkBasicRequirementsAndDisplayLoadingScreen()
     protected function resolveDependencies()
     {
         $objectManager = $this->bootstrap->getObjectManager();
-        $componentChainFactory = $objectManager->get(ComponentChainFactory::class);
         $configurationManager = $objectManager->get(ConfigurationManager::class);
         $this->settings = $configurationManager->getConfiguration(ConfigurationManager::CONFIGURATION_TYPE_SETTINGS, 'Neos.Flow');
         $setupSettings = $configurationManager->getConfiguration(ConfigurationManager::CONFIGURATION_TYPE_SETTINGS, 'Neos.Setup');
-        $httpChainSettings = Arrays::arrayMergeRecursiveOverrule($this->settings['http']['chain'], $setupSettings['http']['chain']);
-        $this->baseComponentChain = $componentChainFactory->create($httpChainSettings);
+        $httpChainSettings = Arrays::arrayMergeRecursiveOverrule($this->settings['http']['middlewares'], $setupSettings['http']['middlewares']);
```

Comment:
Not really sure why that manual config merging shenanigan is necessary at all?

Reply:
mh, I don't think that this is still needed actually